sycl-fusion/passes/kernel-fusion/SYCLKernelFusion.cpp

```diff
@@ -817,7 +818,7 @@ SYCLKernelFusion::mergeReqdWorkgroupSize(KernelAttr *Attr,
     return SYCLKernelFusion::AttrMergeResult::AddAttr;
   }
   for (size_t I = 0; I < 3; ++I) {
```

Comment:
I think this loop can be replaced with `Attr->Values != Other.Values`

Reply:
Yes, neat!